src/main/java/com/fnp/Webapp Fundamentals/newcrud/src/main/java/com/fnp/controller/UpdateController.java

```diff
@@ -0,0 +1,32 @@
+package com.fnp.controller;
+
+import java.io.IOException;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.fnp.service.UpdateService;
+
+/**
+ * Update controller for update function.
+ * @author Saransh vijay
+ *
+ */
+@WebServlet("/update")
+public class UpdateController extends HttpServlet {
+
+	private static final long serialVersionUID = 1L;
+
+	@Override
+	protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+
+		/**
+		 * String type of Service class updateOp Business Logic.
+		 */
+		UpdateService.updateOp(req, resp);
```

Comment:
Remove all the static calls to service classes and make these object ref calls. to methods. to do any kind of operation.